iTechArt.Api/Controllers/AirportController.cs

```diff
@@ -44,9 +44,9 @@ public async Task<IActionResult> ImportAirportExcel(IFormFile file)
         /// Controller of Exporting airport data
         /// </summary>
         [HttpGet("get_all")]
-        public async Task<IActionResult> ExportAirportExcel()
+        public async Task<IActionResult> ExportAirportExcel(int pageIndex)
```

Comment:
difference methods in airport and grocery, what way is right?

Reply:
async method, by name doesn`t contain 'Async' postfix

Reply:
Changed all task returning methods into Async()